include/mbedtls/mbedtls_config.h

```diff
@@ -3651,8 +3666,8 @@
 
 /* To Use Function Macros MBEDTLS_PLATFORM_C must be enabled */
```

Comment:
Not really in scope, but there's a problem here: this sentence won't appear in the doxygen documentation.

Reply:
Also, the grammar is a bit weird (unnecessary capitals). And does it mean

`For the following function macros to be used, MBEDTLS_PLATFORM_C must be enabled` or

`For the prior function macros to be used, MBEDTLS_PLATFORM_C must be enabled`?

Reply:
> Not really in scope, but there's a problem here: this sentence won't appear in the doxygen documentation.

Fortunately we have a check in `check_config.h` to cover that, so it's not a silent failure. 
Would you like me to add an information about it to the `brief` part of the file? I'm not sure if there's a better place (assuming we won't add that information to each macro description).
Also, doxygen won't show the documentation for an undefined macro.

Reply:
> Would you like me to add an information about it to the brief part of the file? I'm not sure if there's a better place (assuming we won't add that information to each macro description).

Ideally to each macro description. But I am sympathetic to the fact that this is getting far out of the original scope. In this pull request, all I'm requesting is that the new documentation should not be wrong, and if it's incomplete, it should not be misleading. For example just saying “you must enable X” is correct but misleading if you actually need to enable both X and Y.

> doxygen won't show the documentation of an undefined macro.

We render the documentation at https://mbed-tls.readthedocs.io/projects/api/en/development/ (formerly on https://tls.mbed.org/) by enabling (almost) every configuration macro.